src/middlewared/middlewared/plugins/acme_protocol.py

```diff
@@ -115,10 +115,7 @@ def do_create(self, data):
         # 3) SAVE REGISTRATION OBJECT
         # 4) SAVE REGISTRATION BODY
 
-        self.middleware.call_sync('network.general.will_perform_activity', 'acme')
```

Comment:
Why?

Reply:
Because `self.get_directory()` makes the exact same call so this was unnecessary.